### ISSUE TITLE
minimal backup only backs-up vis.0 adapter

### ISSUE THREAD

**Issue by daniel-lo** [simatec/ioBroker.backitup#38]
Hi, my minimal backup is incomplete .. it only backs-up vis.0. Full backup seems to be good.

**Comment by peoples0815** [comment]
Did you try the original Backup Command via Shell if it is the same? Because the minimal Backup just executes this command.

**Comment by daniel-lo** [comment]
yes .. I did .. same problem there. posted the problem in ioBroker Forum .. nobody was interested yet.
So I realized that it is not your adapter's fault.

**Comment by simatec** [comment]
Is your System up2date?

**Comment by daniel-lo** [comment]
IMO yes .. but what exactly do you have in mind that could be outdated?
Node.js: v8.12.0
NPM: 4.6.1

**Comment by simatec** [comment]
All Adapter and js-controller

**Comment by daniel-lo** [comment]
yes .. I update immediately when something is new .. js-controller is node.js?

**Comment by simatec** [comment]
No js-Controller is host

**Comment by daniel-lo** [comment]
sorry .. to fast for me .. how do I check if js-controller is up-to-date? I see no adapter called host.

**Comment by simatec** [comment]
You have a button host.
On host you see the Controller Version

**Comment by daniel-lo** [comment]
Thanks .. I never have that showing .. I'm also on 1.4.2
